features/verifying_doubles/partial_doubles.feature

```diff
@@ -16,12 +14,6 @@ Feature: Partial doubles
         "saved!" if user.save
       end
 
-      RSpec.configure do |config|
-        config.mock_with :rspec do |mocks|
-          mocks.verify_partial_doubles = true
-        end
-      end
```

Comment:
Do we want an example with this turned off?

Reply:
Added 👍